File: MAP/JavaRepo - Toy language/Lab11/src/utils/FileData.java
package utils;

import java.io.BufferedReader;

public class FileData {
    private String filename;
    private BufferedReader header;

    public FileData(String m, BufferedReader b){
        filename = m;
        header = b;
    }

    public String getFilename() {
        return filename;
    }

    public void setFilename(String filename) {
        this.filename = filename;
    }

    public BufferedReader getHeader() {
        return header;
    }

    public void setHeader(BufferedReader header) {
        this.header = header;
    }

    @Override
    public String toString(){
        return filename;
    }
}

File: Year 2/Semester 1/MAP/C# - Toy language/Lab10/Toy Language/Toy Language/utils/FileData.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Toy_Language.utils
{
    public class FileData
    {
        private string fileName;
        private StreamReader header;

        public FileData(string _fileName, StreamReader _header)
        {
            FileName = _fileName;
            Header = _header;
        }

        public string FileName { get => fileName; set => fileName = value; }
        public StreamReader Header { get => header; set => header = value; }
            
        public override string ToString()
        {
            return fileName;
        }
    }
}
